src/organizations/interfaces/update-organization-options.interface.ts

```diff
@@ -1,12 +1,22 @@
+import { DomainData } from './domain-data.interface';
+
 export interface UpdateOrganizationOptions {
   organization: string;
   name: string;
   allowProfilesOutsideOrganization?: boolean;
+  /**
+   * @deprecated Use `domain_data` instead.
+   */
   domains?: string[];
+  domain_data?: DomainData[];
```

Comment:
We should use camel-case for parameters to SDK methods, and convert to snake-case as part of the serialization process.